packages/test-kit/src/with-feature.ts

```diff
@@ -255,43 +260,70 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
                 throw new Error('Engine HTTP server is closed!');
             }
 
-            const runningFeature = await executableApp.runFeature({
-                featureName,
-                configName,
-                runtimeOptions: runOptions,
-                overrideConfig: config,
-            });
+            let runningFeature;
```

Comment:
type?

Reply:
also, this pattern of overriding a local variable leaves a lot of room for errors. use an array/set of running features